docs/spec/State-Machine-Behavior-Elements/State-Entry-Specifiers.adoc

```diff
@@ -0,0 +1,31 @@
+=== State Entry Specifiers
+
+A *state entry specifier* is part of a
+<<State-Machine-Behavior-Elements_State-Definitions,state definition>>.
+It specifies the state's entry actions
+
+==== Syntax
+
+`entry` <<State-Machine-Behavior-Elements_Do-Expressions,_do-expression_>>
+
+==== Semantics
+
+. The _do-expression_ specifies a list of entry actions that get executed
```

Comment:
Let's not italicize or hyphenate references to syntactic elements in the explanatory text.
* _do-expression_ is a nonterminal symbol appearing in the grammar. It represents a do expression.
* To refer to a do expression, just say "do expression."

Reply:
Fixed